src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      class="BigSearchForm" 
+      action="#" 
+      method="post"
+      data-qa="big"
+    >
+      <label class="BigSearchForm__label">
```

Comment:
```suggestion
      <label class="bigSearchForm__label">
```
classes should start with lowercase letters